Objects/abstract.c

```diff
@@ -168,6 +169,25 @@ PyObject_GetItem(PyObject *o, PyObject *key)
                               "be integer, not '%.200s'", key);
     }
 
+    if (PyType_Check(o)) {
+        meth = PyObject_GetAttrString(o, "__class_getitem__");
```

Comment:
Is it worth to use `_Py_IDENTIFIER` (here or in other cases)?

Reply:
I switched to identifiers in most cases (I have heard they are faster).